7.figures/themes.r

```diff
@@ -0,0 +1,51 @@
+library(ggplot2)
+library(dplyr)
+
+focus_phenotypes <- c(
```

Comment:
Yes, this would be ideal. However, given our current manuscript state, I recommend that we skip adding a comment here for now, since our focus may shift slightly. We will definitely describe our rationale in the manuscript.